pkgs/development/python-modules/pymc3/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchurl
+, buildPythonPackage
+, Theano
+, gfortran
+, openblas
+, numpy
+, scipy
+, matplotlib
+, h5py
+, pandas
+, patsy
+, tqdm
+, joblib
+}:
+
+buildPythonPackage rec {
+  pname = "pymc3";
+  version = "3.2";
+
+  propagatedBuildInputs = [
+    Theano
+    gfortran
+    openblas
+    numpy
+    scipy
+    matplotlib
+    # unzip
+    h5py
+    pandas
+    patsy
+    tqdm
+    joblib
+  ];
```

Comment:
several packages you mentioned here are dependencies of others. No need to repeat them, although it also doesn't hurt.